README.md

```diff
@@ -20,6 +20,29 @@ npm run build
 npm run lint
 ```
 
+## How to write Articles?
+Articles are stored as JSON files in ```src/data/articles```. To add a new article you can add a new JSON file.
+Also add your new article in the file ```src/data/articles/articleContents```. This will make them appear
+in the article overview (Wiki) and in the search.
+
+#### Article Structure
+An article contains of a ```title```, ```paragraph```, ```image``` and ```chapters```. The image is optional.
+Look into other articles and their JSON's for examples.
+
+A chapter contains of ```subtitle```, ```paragraph``` and ```image```. The image is optional.
```

Comment:
Grammar stuff:

I just had to google this usage of ``contains of`` 😄 learned someting new in English, thanks.
I think we should be consistent though, above you wrote ``contains of a`` and here you omit the ``a``.